cmd/promscale/main.go

```diff
@@ -14,7 +14,7 @@ import (
 
 func main() {
 	cfg := &runner.Config{}
-	cfg, err := runner.ParseFlags(cfg)
+	cfg, err := runner.ParseFlags(cfg, os.Args[1:])
```

Comment:
I couldn't get the need for this, so if you can tell that.

Reply:
The need is to have arguments passed into the function so we can use them for testing purposes. Without this, the function is very hard to test and its getting more complicated so it needs testing IMO.